src/scheduler/SchedulerSubSystem.java

```diff
@@ -0,0 +1,92 @@
+package scheduler;
+
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.SocketException;
+
+import elevator.Elevator;
+import elevator.ElevatorQueue;
+import floor.SimulationVars;
+import messages.ElevatorMessage;
+import messages.ElevatorMessage.MessageType;
+import messages.ElevatorRequestMessage;
+import messages.ElevatorRequestMessage.Direction;
+import messages.FloorArrivalMessage;
+import messages.FloorRequestMessage;
+import messages.Message;
+
+public class SchedulerSubSystem {
+	private Scheduler scheduler;
+	private SchedulerMessenger messenger;
+	private FaultDetector faultDetector;
```

Comment:
We dont need `faultDetector` to be an instance variable, it's only used once in the constructor.